.github/workflows/dotnet.yml

```diff
@@ -7,6 +7,9 @@ on:
     branches: [ '**' ]
   # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
+  
+env:
+    NUGET_VERSION: 3.0.0
```

Comment:
Two short comments about this version number:
* In principle we're working towards a version 2.2.0 right now. So you are suggesting to bump it directly to 3.0 with the .NET 6 changes?
* Currently we use the output of `git describe` for versioning the nupkgs built via GHA. This has the advantage that it gives better distinction of off-release commits on master, and versioning is essentially controlled via tags (i.e. we get a version 3.0 as soon as we set the corresponding tag).

Reply:
i would put out 2.2.0 first and then 3.0 as a beta probably